modules/php/Models/Player.php

```diff
@@ -518,4 +518,27 @@ public function react($ids, $ctx)
       }
     }
   }
+
+  public function prepareSelection($card, $selectionCards, $playerIds)
+  {
+    $src = $card->getName();
+    $atom = [
+      'state' => ST_SELECT_CARD,
+//      'type' => 'selection_draw' // TODO: Do we actually need 'type'?
```

Comment:
That's a good question, maybe for some weird corner cases, but probably not for selection indeed

Reply:
Deleted then